PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -65,8 +88,9 @@ can be found in
 
     dspace/config/modules/swordv2-server.cfg
     
-By default it will crosswalk a broad selection of Dublin Core fields, and you can customise that here.  If you wish to also support RIOXX you may need to
-extend your EntryIngester implementation.
+By default it will crosswalk a broad selection of Dublin Core fields, and you can customise that here. [More info](https://wiki.duraspace.org/display/DSDOC5x/Metadata+and+Bitstream+Format+Registries)
```

Comment:
Change: `.... Dublin Core fields, but this may be customised.  See [here] (...) for more information.`